.gitignore

```diff
@@ -69,6 +69,8 @@ core/www/
 core/test-results/
 core/playwright-report/
 core/**/*-snapshots
```

Comment:
All screenshots are currently ignored locally. Do we need to reconsider this regex pattern to allow screenshots generated from Linux to be tracked by source control?

I can see this being a point of confusion for developers, that they would need to forcibly track the new screenshots.

Reply:
Good catch - fixed. Tested this locally by making a change to chip and screenshots updated inside of Docker are tracked correctly. Screenshots updated outside of Docker are not tracked.

![image](https://github.com/ionic-team/ionic-framework/assets/2721089/d5135dd9-cfa1-459f-a458-65c1752c941e)